src/iblt.cpp

```diff
@@ -117,6 +138,13 @@ void CIblt::resize(size_t _expectedNumEntries)
 
     CIblt::n_hash = OptimalNHash(_expectedNumEntries);
 
+    if (salt > VALS_32 / n_hash)
+        throw std::runtime_error("salt * n_hash must fit in uint32_t");
```

Comment:
do we really need this, or can mapHashIdxSeeds wrap around?

Reply:
Removed in 4eee6a1.